app/src/main/res/layout/view_simple_alert_sheet.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/background_bottom_sheet"
+    android:orientation="vertical"
+    android:padding="24dp"
+    app:layout_behavior="android.support.design.widget.BottomSheetBehavior">
+
+    <ImageView
+        android:id="@+id/icon"
+        style="@style/AlertSheetImageStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="114dp"
```

Comment:
this breaks when trying to use any image other than one sized like the IG reminder one @hackertronix . I'll open a JIRA ticket 👍